pushing different pricing models bite (#366)
panoptic-labs/docs: research/2025-08-04-pricing-models-perp-options/pricing-models-perp-options.md

---
slug: pricing-models-perp-options
title: "How to Price Perpetual Options: Five Models Compared"

> tags: [Pricing Model, Black-Scholes, American Option]

Reviewer: Remove the 'dash' in Black-Scholes so that it conforms with other articles that use this tag. "Black Scholes"

image: /img/research/pricing-models-perp-options-banner.png

> description: "This paper presents a rigorous overview of several perpetual option pricing frameworks and compares them to the Panoptic protocol, an oracle-free, DeFi-native options model built on Uniswap v3 and v4. We highlight the mathematical foundations of each model, discuss their pricing mechanics, and analyze their applicability in decentralized markets."

Reviewer: Description is too long. Need to shorten to one sentence. Something like: "This paper compares perpetual option pricing frameworks with Panoptic’s oracle-free, DeFi-native model on Uniswap v3/v4, detailing mathematical foundations, pricing mechanics, and applicability to decentralized markets."

- Option holders pay a fee based on the time spent within the LP range and actual trading volume.

**Streaming Premium Mechanism:**

The core idea is that Uniswap LPs accumulate fees when the price stays near their tick range — especially when liquidity is concentrated, this payoff resembles short option exposure. Panoptic lets users take the long side by paying a fee proportional to:

$$
\text{Premium} = \int_0^T \text{FeeRate}(S_t, L, U, \text{Volume}_t) \cdot \mathbb{1}_{\{S_t \in [L, U]\}} \, dt
$$

Where:

- $S_t$ is the spot price of the underlying asset at time $t$, as observed directly from Uniswap v3 or v4 ticks.
- $L$ is the lower bound of the LP’s tick range (acts like the strike floor).
- $U$ is the upper bound of the LP’s tick range (acts like the strike ceiling).
- $T$ is the total duration of the option position (not expiry, but how long the trader holds the position).

The indicator function $\mathbb{1}_{\{S_t ∈ [L, U]\}}$ ensures that fees are only accrued while the spot price is within the specified LP range.

The protocol defines a function `FeeRate` based on:

- Tick spacing (width of range),
- Volume traded inside the range,
- Amount of base liquidity,

> - Historical fees collected.

Reviewer: ```suggestion
```

Reviewer: remove this bullet point (FeeRate just depnds on the first three, not on past fees collected)

The result is a set of clear formulas:

- If the asset price is below a certain boundary, you wait.
- If it's above, you exercise the outer option, claim the inner one, and then you're in the game.

Each compound structure — whether it’s a call-on-put, chooser, or put-on-call — has its own logic and boundary. But the idea is always the same: wait until it’s worth unlocking the next layer.

**Key difference from Panoptic:**  
While Panoptic relies on continuous fee accrual driven by path-dependent Uniswap activity, the Gapeev & Rodosthenous model assumes no trading activity and instead solves for closed-form optimal exercise boundaries in a frictionless market. Panoptic options are paid as-you-go and can be held indefinitely, while this compound model requires an explicit stopping decision at well-defined price levels. Unlike Panoptic’s emergent pricing from market mechanics, this model is fully analytical and assumes complete market observability.

## Model V: Nonlinear Volatility in Perpetual American Put Options (Grossinho, Faghan, & Ševčovič, 2017)

### Quantitative Explanation

> This model extends the classical Black-Scholes framework for pricing perpetual American put options by allowing the volatility function \( \sigma \) to depend nonlinearly on the second derivative (Gamma) of the option price.

Reviewer: ```suggestion
This model extends the classical Black-Scholes framework for pricing perpetual American put options by allowing the volatility function $\sigma$ to depend nonlinearly on the second derivative (Gamma) of the option price.
```

| **On-Chain Compatibility** | Fully native (built on AMM tick data) | Difficult; oracle and off-chain needed | Medium; numerical methods needed | Difficult; compound logic not natively modular | Challenging; requires nonlinear PDE solver |
| **Greeks View** | Theta-based revenue, gamma bounded by LP range | Classic delta/gamma model; theta funds flow | Average Greeks across expiries | Multiple Greeks layered; sensitive to boundary rules | Volatility is a function of gamma; nonlinearity impacts all Greeks |
| **Innovation** | Oracle-free, real-liquidity derived perpetual options | Replicates option decay via perps-style funding | Elegant link between perpetual options and vanilla blends | First closed-form perpetual compound American options | Introduces feedback-sensitive volatility into perpetual puts |
| **Main Limitation** | No closed-form price; requires simulation; high path variance | Dependent on oracle accuracy; off-chain components | Ignores real-time path; more theoretical than practical | Complex to implement and nest in composable DeFi | Hard to solve analytically; parameter calibration is complex |

> **Comparative table of perpetual option pricing models**

Reviewer: ```suggestion
```

## Conclusion

Perpetual options represent more than an innovation in pricing, they signal a shift in how we design financial primitives. Without expiry, the boundaries of flexibility, composability, and on-chain autonomy expand. The challenge is no longer how to replicate Wall Street on-chain, but how to invent new forms of risk, flow, and strategy that are native to decentralized markets. Pricing becomes a question of incentive alignment, liquidity behavior, and protocol dynamics. As the ecosystem evolves, perpetual options may power the next generation of credit, leverage, insurance, and structured products; all live, modular, and governed by code. The future of derivatives is not just continuous. It is programmable.

## References

- **Panoptic Protocol Team**  
  *Panoptic Whitepaper v1.3.1*. Published in 2023.  
  https://docs.panoptic.xyz/

- **Dan Robinson and Sam Bankman-Fried**  
  *Everlasting Options*. Paradigm Research, 2021.  
  https://www.paradigm.xyz/2021/05/everlasting-options

- **Sidani, W.**  

>   *Pricing Perpetual Options Using a Distribution over Expiry*.  
>   (Private draft/notes, circulated in DeFi research community), 2023.

Reviewer: Is there a published paper we can link? Maybe this one:
```suggestion
  *Pricing Perpetual Options Using a Distribution over Expiry*. Published in 2025. 
  https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5065014
```

- **Gapeev, P. V., and Rodosthenous, N.**  
  *Perpetual American Compound Option Valuation*.  
  Quantitative Finance, Volume 13, Issue 1, 2013, pp. 129–139.  

>   https://doi.org/10.1080/14697688.2011.647972

Reviewer: This link is broken. Did you mean:
```suggestion
  https://papers.ssrn.com/sol3/papers.cfm?abstract_id=1698264
```

- **Grossinho, M. R., Faghan, M., and Ševčovič, D.**  
  *Pricing Perpetual Put Options by the Black–Scholes Equation with a Nonlinear Volatility Function*.  
  arXiv preprint arXiv:1611.00885, 2017.  
  https://arxiv.org/abs/1611.00885

- **Ekström, E., and Lu, B.**  
  *Perpetual American Options with Stochastic Volatility*.  
  Mathematical Finance, Volume 31, Issue 2, 2021, pp. 757–785.  
  https://doi.org/10.1111/mafi.12271

> Join the growing community of Panoptimists and be the first to hear our latest updates by following us on our [social media platforms](https://linktr.ee/panopticxyz). To learn more about Panoptic and all things DeFi options, check out our [docs](/docs/intro) and head to [our website](https://panoptic.xyz/).

Reviewer: Italicize
```suggestion
_Join the growing community of Panoptimists and be the first to hear our latest updates by following us on our [social media platforms](https://linktr.ee/panopticxyz). To learn more about Panoptic and all things DeFi options, check out our [docs](/docs/intro) and head to [our website](https://panoptic.xyz/)._
```
